docs/src/app/components/pages/components/DatePicker/ExampleInternational.jsx

```diff
@@ -15,15 +17,19 @@ if (areIntlLocalesSupported('fr')) {
 }
 
 const DatePickerExampleInternational = () => (
-  <DatePicker
-    hintText="fr version"
-    DateTimeFormat={DateTimeFormat}
-    // Intl is supported by most modern browsers, see http://caniuse.com/#search=intl
-    // for browsers that don't support it use this polyfill https://github.com/andyearnshaw/Intl.js
-    wordings={{ok: 'OK', cancel: 'Annuler'}}
-    firstDayOfWeek={1}
-    locale="fr"
-  />
+  <div>
+    <DatePicker
+      hintText="fr version"
+      DateTimeFormat={DateTimeFormat}
+      wordings={{ok: 'OK', cancel: 'Annuler'}}
+      locale="fr"
+    />
+    <DatePicker
+      hintText="en-US version"
+      formatDate={(date) => date.toLocaleDateString()}
```

Comment:
But I can use your DateTimeFormat example as a safer alternative to date.toLocaleDateString().